career-mentorship-program/2021-spring/matching_code.py

```diff
@@ -239,44 +318,48 @@ def from_survey_response(row):
         mentor = Mentor(student)
 
         mentor.experienced_fields = frozenset(row[MENTOR_EXPERIENCED_FIELDS].split(";"))
```

Comment:
See comments on the corresponding mentee code and apply the same suggestions to this section!